<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analyzer::gui::main_window</class>
 <widget class="QMainWindow" name="analyzer::gui::main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <layout class="QHBoxLayout" name="tag_layout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="frame_display">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>No Sequence Loaded</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSpinBox" name="frame_spinbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="prefix">
           <string>Frame </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="frame_slider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="sequence_combobox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Select the sequence to display.</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="load_dataset_button">
        <property name="toolTip">
         <string>Load a tracking dataset.</string>
        </property>
        <property name="text">
         <string>Dataset</string>
        </property>
        <property name="icon">
         <iconset theme="folder-open">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+Shift+O</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="bg_candidates_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Background Candidates</string>
   </property>
   <property name="toolTip">
    <string>Draw background candidate data in the graph.</string>
   </property>
  </action>
  <action name="bg_mined_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Background Mined</string>
   </property>
   <property name="toolTip">
    <string>Draw mined background data in the graph.</string>
   </property>
  </action>
  <action name="tg_candidates_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Target Candidates</string>
   </property>
   <property name="toolTip">
    <string>Draw target candidate data in the graph.</string>
   </property>
  </action>
  <action name="thresholds_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Thresholds</string>
   </property>
   <property name="toolTip">
    <string>Draw the threshold lines in the graph.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sequence_combobox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>analyzer::gui::main_window</receiver>
   <slot>change_sequence(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>974</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>974</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_dataset_button</sender>
   <signal>clicked()</signal>
   <receiver>analyzer::gui::main_window</receiver>
   <slot>load_dataset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1030</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>1031</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frame_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>analyzer::gui::main_window</receiver>
   <slot>change_frame(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frame_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>analyzer::gui::main_window</receiver>
   <slot>change_frame(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>735</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bg_candidates_action</sender>
   <signal>toggled(bool)</signal>
   <receiver>analyzer::gui::main_window</receiver>
   <slot>toggle_bg_candidate_plot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bg_mined_action</sender>
   <signal>toggled(bool)</signal>
   <receiver>analyzer::gui::main_window</receiver>
   <slot>toggle_bg_mined_plot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tg_candidates_action</sender>
   <signal>toggled(bool)</signal>
   <receiver>analyzer::gui::main_window</receiver>
   <slot>toggle_tg_candidate_plot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholds_action</sender>
   <signal>toggled(bool)</signal>
   <receiver>analyzer::gui::main_window</receiver>
   <slot>toggle_thresholds_plot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>load_tracking_data()</slot>
  <slot>save_graph()</slot>
  <slot>change_update(int)</slot>
  <slot>change_training_batch(int)</slot>
  <slot>change_sequence(int)</slot>
  <slot>load_dataset()</slot>
  <slot>change_frame(int)</slot>
  <slot>change_point_size(int)</slot>
  <slot>toggle_bg_candidate_plot(bool)</slot>
  <slot>toggle_bg_mined_plot(bool)</slot>
  <slot>toggle_tg_candidate_plot(bool)</slot>
  <slot>toggle_thresholds_plot(bool)</slot>
 </slots>
</ui>
